### PR TITLE
add 4.12 e2e tests for aws-ipi-sdn-workers-rhel8 related

### PR DESCRIPTION
**Pull request by lihongan**
follow up https://github.com/openshift/release/pull/31307

### CODE REVIEW

**Comment by lihongan** [comment]
@liangxia @zhaozhanqi @anuragthehatter PTAL. thank.

**Comment by liangxia** [comment]
There are conflicts

**Comment by anuragthehatter** [comment]
/lgtm

thanks!

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 periodics:
 - agent: kubernetes
-  cluster: build03
+  cluster: build01
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml:3]: Are those changes expected ?

Comment (lihongan) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml]: Just did rebase and make jobs, no idea why it happened.
Redo and it looks good now.

**Pull request merged**